File: Week-1/2-mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Read Heigh
    int h;
    
    do 
    {
        h = get_int("Height: ");
    } 
    while (h < 1 || h > 8);
    
    // Build pyramide
    for (int i = 0; i < h; i++)
    {
        int j;
        // Empty spaces #1
        for (j = i; j < h - 1; j++)
        {
            printf(" ");  
        }
            
        // Bricks #1
        for (int k = j; k < h + i; k++)
        {
            printf("#"); 
        }
        
        // Gap
        printf("  ");
            
        // Bricks #2
        for (int k = j; k < h + i; k++)
        {
            printf("#"); 
        }
        
        // Next line 
        printf("\n");
    }
}

File: Week-6/1 - mario-less/mario.py
# Import get_int method
from cs50 import get_int
# Read user input
while True:
    height = get_int("Height: ")
    if height > 0 and height <= 8:
        break
# Build a half-pyramid
for i in range(height):
    for j in range(height - i - 1):
        print(" ", end="")
    for j in range(i + 1):
        print("#", end="")
    print()